bind/config.sls

```diff
@@ -196,7 +209,7 @@ bind_rndc_client_config:
 {% if file and zone_data['type'] == 'master' -%}
 zones{{ dash_view }}-{{ zone }}{{ '.include' if serial_auto else ''}}:
   file.managed:
-    - name: {{ map.named_directory }}/{{ file }}{{ '.include' if serial_auto else ''}}
+    - name: {{ map.named_directory }}/{{ map.zones_directory }}/{{ file }}{{ '.include' if serial_auto else ''}}
```

Comment:
@ryanwalder, @joe-bowman having a double-variable parameter makes it impossible to use a directory **outside** of `{{ map.named_directory }}` to store zone files. If I want to have all the configuration files under `{{ map.named_directory }}` (say **/var/cache/bind** or **/var/named**) and the zone files under `{{ map.zones_directory }}` (say **/data/zones**), this way is impossible.

That's why I suggested either setting it as an **independent, full path variable**, ie:

```yaml
zones_directory: /var/cache/bind/zones
```
and use it like

```yaml
zones{{ dash_view }}-{{ zone }}{{ '.include' if serial_auto else ''}}:
  file.managed:
    - name: {{ map.zones_directory }}/{{ file }}{{ '.include' if serial_auto else ''}}
...
```

Reply:
That's a good point, I'll sort that today. Unfortunately I had made the changes before you made your comment!